gradio/external.py

```diff
@@ -538,9 +538,7 @@ def from_spaces_blocks(space: str, hf_token: str | None) -> Blocks:
     predict_fns = []
     for fn_index, endpoint in enumerate(client.endpoints):
         assert isinstance(endpoint, Endpoint)
-        helper = None
-        if endpoint.protocol in ("ws", "sse"):
-            helper = client.new_helper(fn_index)
+        helper = client.new_helper(fn_index)
```

Comment:
Initially I had added "sse_v1" to the list but actually I don't think there's any reason to have this check because `gr.load()` only works with `gradio==4.x` which includes the protocol anymore

Reply:
I think this makes sense